sirepo/srunit.py

```diff
@@ -6,11 +6,14 @@
 """
 from __future__ import absolute_import, division, print_function
 from pykern import pkcompat
+from pykern import pkjson
 from pykern.pkcollections import PKDict
+from sirepo.pkcli import service
 import flask
 import flask.testing
 import json
 import re
+import requests
```

Comment:
Move this down where it is needed.

Reply:
https://github.com/radiasoft/sirepo/commit/3ebafe869d9a96eef312f68fd903905c1d2b887e